src/js/index.js

```diff
@@ -1009,21 +1009,28 @@ export const hpe = deepFreeze({
     },
     border: {
       side: 'bottom',
-      color: 'border',
+      color: 'transparent',
+      size: 'medium',
       active: {
         color: 'border-strong',
       },
       disabled: {
```

Comment:
Would it work to do `disabled: undefined` or `disabled: {}` or `disabled: { color: undefined }` instead? Just trying to keep the DOM cleaner where possible.

Reply:
I will double check but I believe we needed transparent to maintain the proper dimensions / text alignment. let me try with undefined now though.

Reply:
I've done `disabled: { color: undefined }` in the site locally, and that works.

Reply:
same for hover.